diff --git a/assets/js/course/alerter.js b/assets/js/course/alerter.js
index e118f3f..53f5e82 100644
--- a/assets/js/course/alerter.js
+++ b/assets/js/course/alerter.js
@@ -1,5 +1,3 @@
-
-
 var alerter = function($wrapper) {
     this.initialize($wrapper);
 };
@@ -29,4 +27,4 @@ $.extend(alerter.prototype, {
     }
 });
 
-module.exports = alerter;
+export default alerter;
diff --git a/assets/js/course/app.js b/assets/js/course/app.js
index 6eeeca0..31c43f1 100644
--- a/assets/js/course/app.js
+++ b/assets/js/course/app.js
@@ -1,4 +1,4 @@
-var alerter = require('./alerter');
+import alerter from "./alerter";
 
 jQuery(document).ready(function() {
     var alertApp = new alerter($('#course_edit'));
