diff --git a/app/DoctrineMigrations/Version20160605192834.php b/app/DoctrineMigrations/Version20160605192834.php
new file mode 100644
index 0000000..64d33f5
--- /dev/null
+++ b/app/DoctrineMigrations/Version20160605192834.php
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160605192834 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE student_enrollment CHANGE course_id course_id INT NOT NULL, CHANGE student_id student_id INT NOT NULL');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE student_enrollment CHANGE course_id course_id INT DEFAULT NULL, CHANGE student_id student_id INT DEFAULT NULL');
+    }
+}
diff --git a/src/AppBundle/Entity/StudentEnrollment.php b/src/AppBundle/Entity/StudentEnrollment.php
index f5073b9..c845d25 100644
--- a/src/AppBundle/Entity/StudentEnrollment.php
+++ b/src/AppBundle/Entity/StudentEnrollment.php
@@ -19,11 +19,13 @@ class StudentEnrollment
 
     /**
      * @ORM\ManyToOne(targetEntity="Course")
+     * @ORM\JoinColumn(nullable=false)
      */
     private $course;
 
     /**
      * @ORM\ManyToOne(targetEntity="Student")
+     * @ORM\JoinColumn(nullable=false)
      */
     private $student;
 
@@ -32,22 +34,28 @@ class StudentEnrollment
         return $this->id;
     }
 
+    /**
+     * @return Course
+     */
     public function getCourse()
     {
         return $this->course;
     }
 
-    public function setCourse($course)
+    public function setCourse(Course $course)
     {
         $this->course = $course;
     }
 
+    /**
+     * @return Student
+     */
     public function getStudent()
     {
         return $this->student;
     }
 
-    public function setStudent($student)
+    public function setStudent(Student $student)
     {
         $this->student = $student;
     }
