diff --git a/app/DoctrineMigrations/Version20160605192625.php b/app/DoctrineMigrations/Version20160605192625.php
new file mode 100644
index 0000000..fcd4a68
--- /dev/null
+++ b/app/DoctrineMigrations/Version20160605192625.php
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160605192625 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE student_enrollment (id INT AUTO_INCREMENT NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP TABLE student_enrollment');
+    }
+}
diff --git a/app/DoctrineMigrations/Version20160605192746.php b/app/DoctrineMigrations/Version20160605192746.php
new file mode 100644
index 0000000..e6fff5c
--- /dev/null
+++ b/app/DoctrineMigrations/Version20160605192746.php
@@ -0,0 +1,42 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160605192746 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE student_enrollment ADD course_id INT DEFAULT NULL, ADD student_id INT DEFAULT NULL');
+        $this->addSql('ALTER TABLE student_enrollment ADD CONSTRAINT FK_36033A00591CC992 FOREIGN KEY (course_id) REFERENCES course (id)');
+        $this->addSql('ALTER TABLE student_enrollment ADD CONSTRAINT FK_36033A00CB944F1A FOREIGN KEY (student_id) REFERENCES student (id)');
+        $this->addSql('CREATE INDEX IDX_36033A00591CC992 ON student_enrollment (course_id)');
+        $this->addSql('CREATE INDEX IDX_36033A00CB944F1A ON student_enrollment (student_id)');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE student_enrollment DROP FOREIGN KEY FK_36033A00591CC992');
+        $this->addSql('ALTER TABLE student_enrollment DROP FOREIGN KEY FK_36033A00CB944F1A');
+        $this->addSql('DROP INDEX IDX_36033A00591CC992 ON student_enrollment');
+        $this->addSql('DROP INDEX IDX_36033A00CB944F1A ON student_enrollment');
+        $this->addSql('ALTER TABLE student_enrollment DROP course_id, DROP student_id');
+    }
+}
diff --git a/src/AppBundle/Entity/StudentEnrollment.php b/src/AppBundle/Entity/StudentEnrollment.php
new file mode 100644
index 0000000..f5073b9
--- /dev/null
+++ b/src/AppBundle/Entity/StudentEnrollment.php
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="student_enrollment")
+ */
+class StudentEnrollment
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Course")
+     */
+    private $course;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Student")
+     */
+    private $student;
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function getCourse()
+    {
+        return $this->course;
+    }
+
+    public function setCourse($course)
+    {
+        $this->course = $course;
+    }
+
+    public function getStudent()
+    {
+        return $this->student;
+    }
+
+    public function setStudent($student)
+    {
+        $this->student = $student;
+    }
+}
