diff --git a/assets/js/course/alerter.js b/assets/js/course/alerter.js
index d193f09..e118f3f 100644
--- a/assets/js/course/alerter.js
+++ b/assets/js/course/alerter.js
@@ -1,6 +1,10 @@
 
 
-var alerter = {
+var alerter = function($wrapper) {
+    this.initialize($wrapper);
+};
+
+$.extend(alerter.prototype, {
     initialize: function($wrapper) {
         this.$wrapper = $wrapper;
 
@@ -23,6 +27,6 @@ var alerter = {
     sendAlert: function(msg) {
         alert(msg);
     }
-};
+});
 
 module.exports = alerter;
diff --git a/assets/js/course/app.js b/assets/js/course/app.js
index 7eb6d6e..6eeeca0 100644
--- a/assets/js/course/app.js
+++ b/assets/js/course/app.js
@@ -1,5 +1,5 @@
 var alerter = require('./alerter');
 
 jQuery(document).ready(function() {
-    alerter.initialize($('#course_edit'));
+    var alertApp = new alerter($('#course_edit'));
 });
