diff --git a/assets/js/course/alerter.js b/assets/js/course/alerter.js
index 641968f..63a58d1 100644
--- a/assets/js/course/alerter.js
+++ b/assets/js/course/alerter.js
@@ -2,7 +2,7 @@
 
 var alerter = {
     initialize: function() {
-        $('#course_edit .js-alert').on('click', this._handleSendAlertClick);
+        $('#course_edit .js-alert').on('click', $.proxy(this._handleSendAlertClick, this));
     },
 
     _handleSendAlertClick: function(e) {
