diff --git a/assets/js/course/Components/CourseApp.react.js b/assets/js/course/Components/CourseApp.react.js
index c26de77..205719f 100644
--- a/assets/js/course/Components/CourseApp.react.js
+++ b/assets/js/course/Components/CourseApp.react.js
@@ -1,10 +1,12 @@
 var React = require('react');
+var StudentList = require('./StudentList.react');
 
 var CourseApp = React.createClass({
   render: function() {
       return (
           <div>
               <h1>{this.props.course.name}</h1>
+                <StudentList />
           </div>
       );
   }
diff --git a/assets/js/course/Components/StudentList.react.js b/assets/js/course/Components/StudentList.react.js
new file mode 100644
index 0000000..e2b1071
--- /dev/null
+++ b/assets/js/course/Components/StudentList.react.js
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var StudentList = React.createClass({
+    render: function () {
+        return (
+            <ul>
+                <li>Test!</li>
+            </ul>
+        );
+    }
+});
+
+module.exports = StudentList;
