diff --git a/assets/js/course/Components/StudentRow.react.js b/assets/js/course/Components/StudentRow.react.js
index b1f3d85..46db87f 100644
--- a/assets/js/course/Components/StudentRow.react.js
+++ b/assets/js/course/Components/StudentRow.react.js
@@ -1,10 +1,15 @@
 var React = require('react');
 
 var StudentRow = React.createClass({
+    getFullName: function() {
+        return this.props.enrollment.firstName+' '+this.props.enrollment.lastName
+    },
+
     render: function () {
         return (
             <tr>
-                <td>I'm a student row!</td>
+                <td>{this.props.enrollment.email}</td>
+                <td>{this.getFullName()}</td>
             </tr>
         )
     }
