diff --git a/assets/webpack.config.js b/assets/webpack.config.js
index c20a32e..028ac45 100644
--- a/assets/webpack.config.js
+++ b/assets/webpack.config.js
@@ -14,7 +14,7 @@ module.exports = {
                 exclude: /node_modules/,
                 loader: "babel-loader",
                 query: {
-                    presets: ['es2015']
+                    presets: ['es2015', 'react']
                 }
             }
         ]
diff --git a/package.json b/package.json
index b41d619..a80f40b 100644
--- a/package.json
+++ b/package.json
@@ -20,8 +20,10 @@
   },
   "homepage": "https://github.com/weaverryan/students_demo#readme",
   "devDependencies": {
+    "babel-cli": "^6.9.0",
     "babel-core": "^6.9.1",
     "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.9.0"
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-react": "^6.5.0"
   }
 }
