diff --git a/assets/js/course/Components/CourseApp.react.js b/assets/js/course/Components/CourseApp.react.js
index c26de77..205719f 100644
--- a/assets/js/course/Components/CourseApp.react.js
+++ b/assets/js/course/Components/CourseApp.react.js
@@ -1,10 +1,12 @@
 var React = require('react');
+var StudentList = require('./StudentList.react');
 
 var CourseApp = React.createClass({
   render: function() {
       return (
           <div>
               <h1>{this.props.course.name}</h1>
+                <StudentList />
           </div>
       );
   }
diff --git a/assets/js/course/Components/StudentList.react.js b/assets/js/course/Components/StudentList.react.js
new file mode 100644
index 0000000..b6e5524
--- /dev/null
+++ b/assets/js/course/Components/StudentList.react.js
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var StudentList = React.createClass({
+    render: function () {
+        return (
+            <table className="table">
+                <thead>
+                    <tr>
+                        <th>Email</th>
+                        <th>Name</th>
+                        <th>&nbsp;</th>
+                    </tr>
+                </thead>
+            </table>
+        );
+    }
+});
+
+module.exports = StudentList;
