diff --git a/assets/js/course/Components/StudentRow.react.js b/assets/js/course/Components/StudentRow.react.js
index 46db87f..9d6efec 100644
--- a/assets/js/course/Components/StudentRow.react.js
+++ b/assets/js/course/Components/StudentRow.react.js
@@ -1,6 +1,12 @@
 var React = require('react');
 
 var StudentRow = React.createClass({
+    handleDeleteClick: function(e) {
+        e.preventDefault();
+
+        alert('clicked!');
+    },
+
     getFullName: function() {
         return this.props.enrollment.firstName+' '+this.props.enrollment.lastName
     },
@@ -10,6 +16,9 @@ var StudentRow = React.createClass({
             <tr>
                 <td>{this.props.enrollment.email}</td>
                 <td>{this.getFullName()}</td>
+                <td>
+                    <a href="#" onClick={this.handleDeleteClick}>X</a>
+                </td>
             </tr>
         )
     }
