diff --git a/src/AppBundle/Entity/StudentEnrollment.php b/src/AppBundle/Entity/StudentEnrollment.php
index c845d25..61caedf 100644
--- a/src/AppBundle/Entity/StudentEnrollment.php
+++ b/src/AppBundle/Entity/StudentEnrollment.php
@@ -3,10 +3,12 @@
 namespace AppBundle\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation as Serializer;
 
 /**
  * @ORM\Entity
  * @ORM\Table(name="student_enrollment")
+ * @Serializer\ExclusionPolicy("all")
  */
 class StudentEnrollment
 {
@@ -14,6 +16,7 @@ class StudentEnrollment
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="AUTO")
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
      */
     private $id;
 
@@ -59,4 +62,12 @@ class StudentEnrollment
     {
         $this->student = $student;
     }
+
+    /**
+     * @Serializer\VirtualProperty()
+     */
+    public function getStudentId()
+    {
+        return $this->getStudent()->getId();
+    }
 }
diff --git a/tests/AppBundle/Controller/CourseControllerTest.php b/tests/AppBundle/Controller/CourseControllerTest.php
index f62113a..92df1ab 100644
--- a/tests/AppBundle/Controller/CourseControllerTest.php
+++ b/tests/AppBundle/Controller/CourseControllerTest.php
@@ -24,5 +24,10 @@ class CourseControllerTest extends ApiTestCase
             $response,
             'id' // id of the enrollment
         );
+        $this->asserter()->assertResponsePropertyEquals(
+            $response,
+            'studentId',
+            $student->getId()
+        );
     }
 }
