diff --git a/src/AppBundle/Controller/Api/StudentController.php b/src/AppBundle/Controller/Api/StudentController.php
new file mode 100644
index 0000000..7a9efdb
--- /dev/null
+++ b/src/AppBundle/Controller/Api/StudentController.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class StudentController extends Controller
+{
+    /**
+     * @Route("/api/students", name="api_student_create")
+     * @Method("POST")
+     */
+    public function createAction()
+    {
+        return new JsonResponse([], 201);
+    }
+}
