diff --git a/tests/AppBundle/Controller/StudentControllerTest.php b/tests/AppBundle/Controller/StudentControllerTest.php
index f9197bc..7e207b4 100644
--- a/tests/AppBundle/Controller/StudentControllerTest.php
+++ b/tests/AppBundle/Controller/StudentControllerTest.php
@@ -2,9 +2,10 @@
 
 namespace Tests\AppBundle\Controller;
 
+use AppBundle\Test\ApiTestCase;
 use GuzzleHttp\Client;
 
-class StudentControllerTest extends \PHPUnit_Framework_TestCase
+class StudentControllerTest extends ApiTestCase
 {
     public function testPOSTCreateStudent()
     {
@@ -14,13 +15,12 @@ class StudentControllerTest extends \PHPUnit_Framework_TestCase
             'lastName' => 'Weaver',
         ];
 
-        $client = new Client(['base_uri' => 'http://localhost:9007']);
-        $response = $client->post('/api/students', [
+        $response = $this->client->post('/api/students', [
             'body' => json_encode($data)
         ]);
 
         $this->assertEquals(201, $response->getStatusCode());
-        $responseBody = json_decode($response->getBody(), true);
-        $this->assertArrayHasKey('id', $responseBody);
+        $this->asserter()
+            ->assertResponsePropertyExists($response, 'id');
     }
 }
diff --git a/web/app_test.php b/web/app_test.php
new file mode 100644
index 0000000..6d168d3
--- /dev/null
+++ b/web/app_test.php
@@ -0,0 +1,32 @@
+<?php
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Debug\Debug;
+
+// If you don't want to setup permissions the proper way, just uncomment the following PHP line
+// read http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup
+// for more information
+//umask(0000);
+
+// This check prevents access to debug front controllers that are deployed by accident to production servers.
+// Feel free to remove this, extend it, or make something more sophisticated.
+if (isset($_SERVER['HTTP_CLIENT_IP'])
+    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
+    || !(in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', 'fe80::1', '::1']) || php_sapi_name() === 'cli-server')
+) {
+    header('HTTP/1.0 403 Forbidden');
+    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
+}
+
+/**
+ * @var Composer\Autoload\ClassLoader $loader
+ */
+$loader = require __DIR__.'/../app/autoload.php';
+Debug::enable();
+
+$kernel = new AppKernel('test', true);
+$kernel->loadClassCache();
+$request = Request::createFromGlobals();
+$response = $kernel->handle($request);
+$response->send();
+$kernel->terminate($request, $response);
