diff --git a/src/AppBundle/Controller/Api/ApiBaseController.php b/src/AppBundle/Controller/Api/ApiBaseController.php
index 88b36b7..94012d0 100644
--- a/src/AppBundle/Controller/Api/ApiBaseController.php
+++ b/src/AppBundle/Controller/Api/ApiBaseController.php
@@ -5,6 +5,8 @@ namespace AppBundle\Controller\Api;
 use JMS\Serializer\SerializationContext;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\Form\Form;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
@@ -37,4 +39,33 @@ abstract class ApiBaseController extends Controller
             'Content-Type' => 'application/json'
         ]);
     }
-}
\ No newline at end of file
+
+    protected function createFailedValidationResponse(Form $form)
+    {
+        $errors = $this->getErrorsFromForm($form);
+
+        return new JsonResponse([
+            'message' => 'Validation failed',
+            'errors' => $errors,
+        ], 400);
+    }
+
+    private function getErrorsFromForm(FormInterface $form)
+    {
+        $errors = array();
+        foreach ($form->getErrors() as $error) {
+            $errors[] = $error->getMessage();
+        }
+
+        foreach ($form->all() as $childForm) {
+            if ($childForm instanceof FormInterface) {
+                if ($childErrors = $this->getErrorsFromForm($childForm)) {
+                    $errors[$childForm->getName()] = $childErrors;
+                }
+            }
+        }
+
+        return $errors;
+    }
+}
+
diff --git a/src/AppBundle/Controller/Api/StudentController.php b/src/AppBundle/Controller/Api/StudentController.php
index 0482e59..b0f29af 100644
--- a/src/AppBundle/Controller/Api/StudentController.php
+++ b/src/AppBundle/Controller/Api/StudentController.php
@@ -25,7 +25,7 @@ class StudentController extends ApiBaseController
         $this->submitForm($request, $form);
 
         if (!$form->isValid()) {
-            throw new BadRequestHttpException();
+            return $this->createFailedValidationResponse($form);
         }
 
         /** @var Student $student */
@@ -48,7 +48,7 @@ class StudentController extends ApiBaseController
         $this->submitForm($request, $form);
 
         if (!$form->isValid()) {
-            throw new BadRequestHttpException();
+            return $this->createFailedValidationResponse($form);
         }
 
         $em = $this->getDoctrine()->getManager();
