diff --git a/src/AppBundle/Entity/Student.php b/src/AppBundle/Entity/Student.php
index 9df0c95..ef695ee 100644
--- a/src/AppBundle/Entity/Student.php
+++ b/src/AppBundle/Entity/Student.php
@@ -3,10 +3,12 @@
 namespace AppBundle\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 
 /**
  * @ORM\Entity
  * @ORM\Table(name="student")
+ * @UniqueEntity(fields={"email"}, message="This email is already taken")
  */
 class Student
 {
